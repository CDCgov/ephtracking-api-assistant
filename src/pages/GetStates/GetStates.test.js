import React from 'react';
import { shallow } from 'enzyme';

import GetStates from './';

describe('Page GetStates', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GetStates />);
    expect(wrapper).toMatchSnapshot();
  });
});
