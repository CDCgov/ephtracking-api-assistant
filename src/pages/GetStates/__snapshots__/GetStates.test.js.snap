// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page GetStates renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GetStates />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h1
          className="title"
>
          Retrieving State FIPS Code for a Measure
</h1>,
        <h5
          className="title is-5"
>
          Usage
</h5>,
        <CodeBlock>
          https://ephtracking.cdc.gov/apigateway/api/
          {
          version
          }
          /getStates/
          {
          measureId
          }
          [?apiToken]
</CodeBlock>,
        <hr />,
        <h5
          className="title is-5"
>
          Set parameters
</h5>,
        <CIM
          handleSelect={[Function]}
/>,
        <hr />,
        <Preview
          url="https://ephtracking.cdc.gov/apigateway/api/v1/getStates/null"
          validUrl={false}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Retrieving State FIPS Code for a Measure",
          "className": "title",
        },
        "ref": null,
        "rendered": "Retrieving State FIPS Code for a Measure",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Usage",
          "className": "title is-5",
        },
        "ref": null,
        "rendered": "Usage",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            "https://ephtracking.cdc.gov/apigateway/api/",
            "{",
            "version",
            "}",
            "/getStates/",
            "{",
            "measureId",
            "}",
            "[?apiToken]",
          ],
        },
        "ref": null,
        "rendered": Array [
          "https://ephtracking.cdc.gov/apigateway/api/",
          "{",
          "version",
          "}",
          "/getStates/",
          "{",
          "measureId",
          "}",
          "[?apiToken]",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Set parameters",
          "className": "title is-5",
        },
        "ref": null,
        "rendered": "Set parameters",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "handleSelect": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "url": "https://ephtracking.cdc.gov/apigateway/api/v1/getStates/null",
          "validUrl": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1
            className="title"
>
            Retrieving State FIPS Code for a Measure
</h1>,
          <h5
            className="title is-5"
>
            Usage
</h5>,
          <CodeBlock>
            https://ephtracking.cdc.gov/apigateway/api/
            {
            version
            }
            /getStates/
            {
            measureId
            }
            [?apiToken]
</CodeBlock>,
          <hr />,
          <h5
            className="title is-5"
>
            Set parameters
</h5>,
          <CIM
            handleSelect={[Function]}
/>,
          <hr />,
          <Preview
            url="https://ephtracking.cdc.gov/apigateway/api/v1/getStates/null"
            validUrl={false}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Retrieving State FIPS Code for a Measure",
            "className": "title",
          },
          "ref": null,
          "rendered": "Retrieving State FIPS Code for a Measure",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Usage",
            "className": "title is-5",
          },
          "ref": null,
          "rendered": "Usage",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              "https://ephtracking.cdc.gov/apigateway/api/",
              "{",
              "version",
              "}",
              "/getStates/",
              "{",
              "measureId",
              "}",
              "[?apiToken]",
            ],
          },
          "ref": null,
          "rendered": Array [
            "https://ephtracking.cdc.gov/apigateway/api/",
            "{",
            "version",
            "}",
            "/getStates/",
            "{",
            "measureId",
            "}",
            "[?apiToken]",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Set parameters",
            "className": "title is-5",
          },
          "ref": null,
          "rendered": "Set parameters",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "handleSelect": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "url": "https://ephtracking.cdc.gov/apigateway/api/v1/getStates/null",
            "validUrl": false,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
